app/controllers/contratos_controller.rb

```diff
@@ -5,18 +5,25 @@ def index
        render json:@contratos
   end
 
+  def values_aleatory ()
```

Comment:
Omit the parentheses in defs when the method doesn't accept any arguments.<br>Do not put a space between a method name and the opening parenthesis.